nvm.sh

```diff
@@ -2957,47 +2977,66 @@ nvm() {
       done
     ;;
     "deactivate")
+      local NVM_USE_SILENT
+      while [ $# -ne 0 ]; do
+        case "$1" in
+          "--silent" | "quiet" | "--quiet" | "silent") NVM_USE_SILENT=1 ;;
```

Comment:
why `quiet` and `--quiet`? i don't think any other part of nvm supports these.

Reply:
Ok, we should stay consistent with the ```nvm use``` command and remove the silent flag too, or would you like the silent flag and the --silent flag to stay for ```nvm use``` and ```nvm deactivate```?

Reply:
I think `--silent` should remain (and eventually be added for every nvm command where it makes sense) - i'm saying i don't think anything supports the word "quiet". In the wider ecosystem, eslint uses that, but most things use "silent", so i'm confused where the impetus to support the term "quiet" comes from.

Reply:
Ok, I will remove ```quiet``` and ```--quiet```. My question was whether I should also add in the ```silent``` flag into the ```nvm use``` command because for consistency with the ```nvm deactivate``` command.

Reply:
ah. I don't think it would make any sense to support "silent" without `--`; it's an option, not a command.
```suggestion
          --silent)
            NVM_USE_SILENT=1
            shift
          ;;
```
also you don't want to use `shift` unless it matches.

Reply:
Sure, makes sense.